doc/common-issues.rst

```diff
@@ -0,0 +1,101 @@
+Frequently Encountered Issues
```

Comment:
Should we link to the troubleshooting TLS errors section of the docs on this page too?